src/test/java/e2e/LoginTest.java

```diff
@@ -16,19 +16,52 @@ public void userCanLogin(){
         loginPage = new LoginPage(app.driver);
         loginPage.login(email,password);
         contactsPage = new ContactsPage(app.driver);
-        Assert.assertTrue(contactsPage.confirmLogin(),"User is not login");
+        Assert.assertTrue(contactsPage.confirmLogin(),"User is not logged");
     }
 
 
     @Test
-    public void userCannotLoginWithInvalidEmail(){
-        String email = "redbullt@gmail.com";
+    public void userCannotLoginWithInvalidEmail() {
+        String email = "redbull@gmail.com";
         String password = "newtest@gmail.com";
 
+        loginPage = new LoginPage(app.driver);
+        loginPage.login(email, password);
+        contactsPage = new ContactsPage(app.driver);
+        Assert.assertTrue(contactsPage.confirmLogin(), "User should not be logged in with invalid email");
```

Comment:
need to use assertFalse for all negative test, after you rollback code in method confirmLogin
because false when we can't view header , true when we can view header , all easy